tutorials/editor/project_manager.rst

```diff
@@ -1,7 +1,7 @@
 .. _doc_project_manager:
 
-Using the Project Manager
-=========================
+Project Manager
```

Comment:
I like this change! IMO sometimes the docs needlessly uses "Using X" rather than just "X" for titles just to follow a guideline.